app/policies/membership_policy.rb

```diff
@@ -7,7 +7,11 @@ def initialize(person, record)
   end
 
   def index?
-    return true if person.memberships.pluck(:status).include?('member')
+    return true if person.staff == true || person.memberships.pluck(:status).include?('member')
+  end
+
+  def approval?
+    index?
```

Comment:
Blocking: This seems flawed. At this time, we should have access to a particular membership, and we want to return false when the person is not a community moderator or a nourish staff member.